ios/FluentUI/Controls/ActivityIndicatorView.swift

```diff
@@ -154,6 +154,15 @@ open class ActivityIndicatorView: UIView {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        if #available(iOS 13, *) {
+            if previousTraitCollection?.userInterfaceStyle != traitCollection.userInterfaceStyle {
+            setupLoaderLayer()
```

Comment:
nit: spacing?

Reply:
Sorry, Swiftlint didn't work on it. Corrected

---
In reply to: [470111446](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470111446) [](ancestors = 470111446)